src/commands/generate.ts

```diff
@@ -1,96 +1,28 @@
 import { GluegunToolbox } from "gluegun"
-import {
-  availableGenerators,
-  generateAppIcons,
-  generateFromTemplate,
-  installedGenerators,
-  showGeneratorHelp,
-  updateGenerators,
-  validateAppIconGenerator,
-} from "../tools/generators"
-import { command, direction, heading, p, warning } from "../tools/pretty"
+import { generateFromTemplate, runGenerator } from "../tools/generators"
+import { command, heading, p, warning } from "../tools/pretty"
 
 module.exports = {
   alias: ["g", "generator", "generators"],
   description: "Generates components and other features from templates",
   run: async (toolbox: GluegunToolbox) => {
-    const { parameters } = toolbox
-
-    p()
-    if (parameters.options.help || parameters.options.list) {
-      // show help or list generators
-      showGeneratorHelp(toolbox)
-    } else if (parameters.options.update) {
-      // update with fresh generators
-      updateGenerators(toolbox)
-    } else if (parameters.first) {
-      // actually generate something
-      generate(toolbox)
-    } else {
-      // catch-all, just show help
-      showGeneratorHelp(toolbox)
-    }
```

Comment:
This is now moved to the `tools/generators.ts` in order to reuse it on the app-icon subcommand